lib/pyenv.rb

```diff
@@ -18,16 +18,18 @@ def initialize(build_wrapper)
 
     def setup!
       install!
+      update!
```

Comment:
@yyuu The updated commit 1b09b98 cleans the nested conditions a bit, however introduces a "dependency" with `require 'set'`.